### DISCUSSION TITLE
Why the Accept button disappear on Mobile?

### DISCUSSION THREAD

**Discussion by alfredovazquez** [manucaralmo/GlowCookies#25] [Q&A]
Hello, is there any reason why the Accept button disappear on Mobile?
And the only way to close the dialog is clicking on the Reject Button :(

![IMG_82E7F80C6479-1](https://user-images.githubusercontent.com/5941449/120785492-7611a480-c52d-11eb-9443-756e0837b456.jpeg)

Thanks in advance :)

**Comment by manucaralmo** [comment]
Hi Alfredo, can you send me your glowcookies script to reproduce the error?
